aea/aea_builder.py

```diff
@@ -823,42 +822,6 @@ def _check_consistent(self, ledger_apis: LedgerApis) -> None:
             ledger_apis.default_ledger_id == self._default_ledger
         ), "Default ledger id of LedgerApis does not match provided default ledger."
 
-    # TODO: remove and replace with a clean approach (~noise based crypto module or similar)
-    def _update_identity(
```

Comment:
@marcofavorito please delete the p2p_noise package entirely from the packages dir

Reply:
Or we can leave it if you feel like leaving it in.

Reply:
But we don't support it so it is probably best deleting it

Reply:
ok